contracts/token/ERC20/RecoverableToken.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity ^0.4.24;
+import "./StandardToken.sol";
+import "../../ownership/Ownable.sol";
+import "./ERC20.sol";
+import "../../AddressUtils.sol";
+
+
+/**
+ * @title Recoverable token
+ * @dev StandardToken modified with recover transfer of other erc20 token.
+ **/
+contract RecoverableToken is StandardToken, Ownable {
```

Comment:
There isn't really any reason to inherit from `StandardToken`, is there? And along that line, I'm not sure `RecoverableToken` is the most appropriate name, since it's not the token that's recoverable, but the contract that allows recovery of _other_ tokens. Maybe something like `RecoversTokens`? I'm not too good at naming things, @shrugs ideas?

Reply:
Like you, I think the name is not perfect @shrugs, for the inherit, I created a new token with this method, partically it herit from standardToken, to have all behaviors of erc20 token

Reply:
Not sure I follow, is your intent to create a token contract that allows for the owner of the contract to retrieve tokens sent to it?